src/js/controller/controller.js

```diff
@@ -773,19 +773,27 @@ Object.assign(Controller.prototype, {
 
         function _seek(pos, meta) {
             const state = _model.get('state');
+            const activeEl = document.activeElement;
+
             if (state === STATE_ERROR) {
                 return;
             }
             if (meta && state === STATE_PLAYING && _model.get('playReason') !== meta.reason) {
                 _model.set('playReason', meta.reason);
             }
+            if (meta && !activeEl.classList.contains('jw-slider-time')) {
```

Comment:
We can seek using the arrow key on the keyboard while the player container is focused, and it should still give you `interaction`. This logic would make that give us `external` instead